domains/tupu.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Tupu4545",
+        "email": "arnish0605@gmail.com"
+    },
+    "record": {
+        "URL": "https://www.youtube.com/watch?v=dqw4w9wgxcq"
```

Comment:
link invalid, also domain will not be released since you are making the domain just to rickroll people